src/main/java/redis/clients/jedis/DefaultJedisClientConfig.java

```diff
@@ -26,9 +26,9 @@ private DefaultJedisClientConfig(int connectionTimeout, int soTimeout, int infin
       String user, String password, int database, String clientName,
       boolean ssl, SSLSocketFactory sslSocketFactory, SSLParameters sslParameters,
       HostnameVerifier hostnameVerifier, HostAndPortMapper hostAndPortMapper) {
-    this.connectionTimeout = connectionTimeout;
-    this.soTimeout = soTimeout;
-    this.infiniteSoTimeout = infiniteSoTimeout;
+    this.connectionTimeoutMillis = connectionTimeout;
```

Comment:
@sazzad16 should you rename the input arguments too?

Reply:
@gkorland yes, I should.